src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.ts

```diff
@@ -1,13 +1,27 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { SidebarCardElement, SidebarTypes } from 'app/types/sidebar';
 
 @Component({
     selector: 'jhi-sidebar-card-item',
     templateUrl: './sidebar-card-item.component.html',
     styleUrls: ['./sidebar-card-item.component.scss', '../sidebar.component.scss'],
 })
-export class SidebarCardItemComponent {
+export class SidebarCardItemComponent implements OnInit {
     @Input() sidebarItem: SidebarCardElement;
     @Input() sidebarType?: SidebarTypes;
     @Input() groupKey?: string;
+    @Input() unreadCount: number = 0;
+
+    formattedUnreadCount: string = '';
+
+    ngOnInit(): void {
```

Comment:
I'm unsure if its due something else, but formatting the number only on initialization of the component means the numbers do not respond to the websocket notifications. Maybe it is possible to react to changes of unreadCount, formatting again and displaying the new number.

Reply:
you're right, thank you. I added the related logic to `ngOnChanges` and I think it solves this issue now.